docs/tutorial/getting-started.rst

```diff
@@ -15,6 +15,10 @@ Prerequisites
 * Ubuntu 22.04 LTS.
 * An Internet connection.
 * `Snap`_ installed.
+* 50GB of disk space is recommended. This includes the `requirements <https://microk8s.io/docs/getting-started>`_ for MicroK8s.
+
+.. note::
+   All the code commands provided throughout this guide can be run from any host terminal.
```

Comment:
Is this correct?

Reply:
The note feels a bit weird to me when I read it initially. What do we want to convey?

Reply:
@kimwnasptd  The place where the user is expected to run the tutorial from. Is that info trivial/assumed? If so, we can remove it.

Thinking, for instance, a non-linux data scientist, coming from Windows, not used to run stuff from a CLI but using UIs.

Reply:
I think in this case it's trivial and would suggest to drop :)